inc/notificationtargetentity.class.php

```diff
@@ -57,11 +59,41 @@ public function addDataForTemplate($event, $options = []) {
       );
       $data = $req->current();
       $this->data['##credit.quantity_remaining##'] = (int)$this->obj->getField('quantity') - (int)$data['consumed_total'];
+      $this->data['##credit.quantity_consumed##'] = (int)$data['consumed_total'];
+
+      $req = $DB->request(
+         [
+            'SELECT' => 'name',
+            'FROM'   => 'glpi_entities',
+            'WHERE'  => [
+               'id' => $this->obj->getField('entities_id'),
+            ]
+         ]
+      );
+      $data = $req->current();
+      $this->data['##credit.entity##'] = $data['name'];
+
+      $req = $DB->request(
+         [
+            'SELECT' => 'name',
+            'FROM'   => 'glpi_plugin_credit_types',
+            'WHERE'  => [
+               'id' => $this->obj->getField('plugin_credit_types_id'),
+            ]
+         ]
+      );
+      $data = $req->current();
+      $this->data['##credit.type##'] = $data['name'];
 
       $this->data['##lang.credit.enddate##'] = __('End date', 'credit');
       $this->data['##lang.credit.quantity_remaining##'] = __('Quantity remaining', 'credit');
       $this->data['##lang.credit.quantity_sold##'] = __('Quantity sold', 'credit');
       $this->data['##lang.credit.name##'] = PluginCreditEntity::getTypeName();
+      $this->data['##lang.credit.overconsumption_allowed##'] = __('Allow overconsumption', 'credit');
+      $this->data['##lang.credit.child_entities##'] = __('Child entities');
```

Comment:
@cedric-anne is it ok to use core translation here (as this string is already translated in core) ?
If I remember correctly you said its not a good idea because the plugin may lose its translations if GLPI's core if updated.

Can you confirm if this should be changed (+ other similar lines below)

Reply:
The problem is that plugin translators are not always reactive, so there is no perfect solution. Reusing GLPI translations is OK, but using GLPI methods to do this is even better, when it is possible (for instance `Entity::getTypeName()` instead of `__('Entity')`.